.env

```diff
@@ -5,3 +5,6 @@ USE_DEVTOOLS=True
 SERVER_WORKERS=1
 QUEUE_WORKERS=1
 LOKOLE_LOG_LEVEL=DEBUG
+STORE_INBOUND_QUEUE_NAME=inbound_store
```

Comment:
Given that the queue names are hardcoded in the `task.py` below, I don't believe they should be configurable here. Seems like a slightly wrong level of abstraction. How about instead having a single environment variable that holds the name of one or more queues which the worker will monitor? (Sort of like what we're doing with the yaml specs when running the API tier.) In this way we can transparently switch between monitoring one or more queues just by updating the environment variable. I'll post an example of what I mean below in my comment on `run-celery.sh`.